iconservice/icon_constant.py

```diff
@@ -52,9 +52,12 @@
 
 BUILTIN_SCORE_ADDRESS_MAPPER = {'governance': "cx0000000000000000000000000000000000000001"}
 
+ZERO_TX_HASH = bytes(32)
+
 REVISION_2 = 2
-REVISION_3 = 3
+REVISION_3 = 3  # 1.1.2.x
 REVISION_4 = 4
+LATEST_REVISION = REVISION_3
```

Comment:
remove REVISION_4

Reply:
we are going to make new task for fix at one time.
so you don't need to fix now